core/modules/system/system.mail.inc

```diff
@@ -20,10 +20,8 @@ class DefaultMailSystem implements MailSystemInterface {
   public function format(array $message) {
     // Join the body array into one string.
     $message['body'] = implode("\n\n", $message['body']);
-    // Convert any HTML to plain-text.
+    // Convert any HTML to plain-text and wrap the mail body for sending.
```

Comment:
Can we please add some more text here to explain things? Here's some text that we can use from the issue description in d.org:

> ...according to the description of [drupal_html_to_text()](http://api.drupal.org/api/drupal/includes%21mail.inc/function/drupal_html_to_text/7), the call to [drupal_wrap_mail()](http://api.drupal.org/api/drupal/includes%21mail.inc/function/drupal_wrap_mail/7) is unnecessary and should be removed since it converts all soft breaks into hard breaks, making format=flowed meaningless.

I thought that we should add something like this:
> Note that backdrop_html_to_text() already calls backdrop_wrap_mail(), so no need to call that explicitly again.